.gitignore

```diff
@@ -40,3 +40,4 @@
 
 **/.rpt2_cache
 .vscode/settings.json
+.idea
```

Comment:
Shouldn't this be part of a global gitignore?

Reply:
Idk, we add the same line for .vscode. Plus then other devs working on this project in IDEA won't have the same problem of files being included in git

Reply:
Personally, I don't have this problem as it's in my global gitignore like suggested by Cezar :shrug: 

I don't think it matters, although, in an ideal world .gitignore should be reserved for byproducts related to the repo.
If we don't add it then let's also remove `.vscode` :stuck_out_tongue: